src/WebJobs.Script/Workers/Http/DefaultHttpWorkerService.cs

```diff
@@ -198,11 +198,11 @@ internal async Task ProcessDefaultInvocationRequest(ScriptInvocationContext scri
                     {
                         if (httpScriptInvocationResult.Outputs == null || !httpScriptInvocationResult.Outputs.Any())
                         {
-                            _logger.LogWarning("Outputs not set on http response for invocationId:{invocationId}", scriptInvocationContext.ExecutionContext.InvocationId);
+                            _logger.LogDebug("Outputs not set on http response for invocationId:{invocationId}", scriptInvocationContext.ExecutionContext.InvocationId);
                         }
                         if (httpScriptInvocationResult.ReturnValue == null)
                         {
-                            _logger.LogWarning("ReturnValue not set on http response for invocationId:{invocationId}", scriptInvocationContext.ExecutionContext.InvocationId);
+                            _logger.LogDebug("ReturnValue not set on http response for invocationId:{invocationId}", scriptInvocationContext.ExecutionContext.InvocationId);
```

Comment:
Is there a way for us to determine whether we're in a case like your extension where return is being set implicitly, and only log a warning if no return is specified using either mechanism?

Reply:
I'm actually not sure why we'd ever want to emit a warning for either of these cases. It's entirely possible that a function returns nothing at all, like in the case of a queue trigger that just executes some code but doesn't use any bindings, right?

Reply:
Agreed - maybe none of these warnings or logs are needed. Would need @pragnagopa to confirm.

Reply:
Warning was initially added for explicitly helping Cx as there is no tooling for Custom Handlers - debug / trace level log is Ok for this.